atlp-rwanda/atlp-pulse-fn
# #DP-108:Fixing URI naming

### PR description 
_Fixing URI naming_
### What this PR does
- Fix login URI name
- Fix organization login URI names
- Remove URIs that are no longer in use
### How can it be tested
Go to the app, and navigate to sognin, the organization login should now be `/organization-login`, and the the login uri should now be `/login`.
### Background context
Successfully merging this PR will introduce new names in the links that are more realistic and easier to understand by any user
### Relevant trello ID
[#DP-108](https://trello.com/c/EQl3uwYN/108-fix-login-uri)
### Screenshot
_N/A_
### Checks completed
